testing/test_fot

```diff
@@ -17,7 +17,7 @@ YQ=venv/bin/yq
 
 echo DAQ spot checks
 
-docker exec forch-faux-4 ping -c10 192.168.1.0
+docker exec forch-faux-4 ping -c10 192.168.1.0 || true
```

Comment:
what's this for?  It's suppressing the test... ?

Reply:
Reverted the change for this line. I didn't notice that this device has a static IP and is not auto-sequstered. So this device will be set to operational state once Forch is started, which means ping is supposed to succeed at this point.